kolibri/plugins/learn/assets/src/vue/breadcrumbs/index.vue

```diff
@@ -1,38 +1,92 @@
 <template>
 
-  <nav class="nav" role="navigation" aria-label="You are here:">
-    <span class="parent">
-      <a v-link="rootLink">Explore</a> 
-    </span>
-    <span class="parent" v-for="crumb in crumbs">
-      <a v-link="crumbLink(crumb.id)">{{ crumb.title }}</a> 
-    </span>
-    <span class="current">
-      <span class="visuallyhidden">Current: </span>
-        <!-- TODO: Get current topic title -->
-    </span>
-  </nav>
+  <div>
+    <nav class="nav" role="navigation" aria-label="You are here:">
+      <span class="parent full-breadcrumbs" v-if="pageMode === $options.PageModes.EXPLORE && pageName !== $options.PageNames.EXPLORE_CONTENT && !isRoot">
+        <a v-link="exploreRoot">Explore</a>
+      </span>
+      <span v-if="pageName === $options.PageNames.LEARN_CONTENT">
+        <a v-link="learnRoot">
+          <span class='sep'>
+            <svg height="24" viewbox="0 0 24 24" width="24" xmlns="http://www.w3.org/2000/svg">
+              <path d="M0 0h24v24H0z" fill="none"></path>
+              <path d="M20 11H7.83l5.59-5.59L12 4l-8 8 8 8 1.41-1.41L7.83 13H20v-2z"></path>
+            </svg>
+          Learn
+          </span>
+        </a>
+      </span>
+      <span v-if="pageName === $options.PageNames.EXPLORE_CONTENT">
+        <a v-link="parentLink">
+          <span class='sep'>
+            <svg height="24" viewbox="0 0 24 24" width="24" xmlns="http://www.w3.org/2000/svg">
+              <path d="M0 0h24v24H0z" fill="none"></path>
+              <path d="M20 11H7.83l5.59-5.59L12 4l-8 8 8 8 1.41-1.41L7.83 13H20v-2z"></path>
+            </svg>
+          Back
+          </span>
+        </a>
+      </span>
+      <span class="parent full-breadcrumbs" v-if="pageMode === $options.PageModes.EXPLORE" v-for="crumb in crumbs">
+        <a v-link="crumbLink(crumb.id)">{{ crumb.title }}</a>
+      </span>
+      <span class="back" v-if="!isRoot && pageMode === $options.PageModes.EXPLORE && pageName !== $options.PageNames.EXPLORE_CONTENT">
```

Comment:
All of the inline `v-if` statements make it somewhat difficult to understand what is supposed to show and when.

Two options for cleaning up the logic:

* move conditions to computed properties, so that the if statements read like simple sentences
* If some of these if statements are mutually exclusive, consider moving the individual breadcrumb styles to sub-components and using a `<component is="...">` strategy, similar to on the root index.vue pages.